Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7992.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 7992, "Datepicker is not opened when we call Datepicker.Focus() in UWP", PlatformAffected.UWP)]
+
+	public class Issue7992 : TestContentPage
+	{
+		protected override void Init()
+		{
+			var stackLayout = new StackLayout();
+			Content = stackLayout;
+
+			stackLayout.Children.Add(new Label { Text = "Label to keep picker from getting initial focus" });
```

Comment:
That's a great point, I just got the issue page to a workable stage to review the method to see if it would be acceptable, so I'll do some more polishing.  I'll also go update the issue page for #7986 with instructions.